### PR TITLE
Clean-up Python #4815 - P1.2: Chem/ChemUtils

### PR DESCRIPTION
**Pull request by IchiruTake**
This is the sub-PR breaking from the large PR #4815

### CODE REVIEW

**Comment by greglandrum** [comment]
Thanks @IchiruTake 
I'd suggest waiting until this and #4859 have been reviewed and merged before submitting any more PRs.

**Review by gedeck — CHANGES_REQUESTED** [review]

File: rdkit/Chem/ChemUtils/AlignDepict.py
Comment (gedeck) [inline rdkit/Chem/ChemUtils/AlignDepict.py:6]: Are you using an automated tool for sorting the imports?

File: rdkit/Chem/ChemUtils/TemplateExpand.py
Comment (gedeck) [inline rdkit/Chem/ChemUtils/TemplateExpand.py:238]: @IchiruTake  See Greg's comment about the second argument.

**Review by gedeck — CHANGES_REQUESTED** [review]

File: rdkit/Chem/ChemUtils/AlignDepict.py
Comment (IchiruTake) [reply rdkit/Chem/ChemUtils/AlignDepict.py:6]: Yes, I used the automatic import sorting in both VSCode and PyCharm, but I fixed this one manually.

File: rdkit/Chem/ChemUtils/TemplateExpand.py
Comment (IchiruTake) [reply rdkit/Chem/ChemUtils/TemplateExpand.py]: Ok, I have accepted the changes, I will put it in my remind list to check

**Review by gedeck — APPROVED** [review]

**Review by greglandrum — APPROVED** [review]

**Pull request merged**
